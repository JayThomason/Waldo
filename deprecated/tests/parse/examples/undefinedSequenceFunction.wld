/** 
  * Do not declare a member of a particular message sequence.  (In
  * this case, Pong.two.)
  *  
  */

PingPong

Endpoint Ping;
Endpoint Pong;


Sequences
{
    PingPongSequence : Ping.one -> Pong.two;
}

Shared
{
    Pong controls Number pongCounter = 0;
}


MessageSequence PingPongSequence
{
    Ping.one(Number someVal)
    {
    }
}


/**** Ping endpoint *********/
Ping
{
}


/**** Pong endpoint *********/
Pong
{
}
