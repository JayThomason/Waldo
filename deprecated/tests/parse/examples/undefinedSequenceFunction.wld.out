Behram test to fix: should throw a more reasonable error.  Essentially was 
trying to look for message receive function, but that wasn't defined.
Pretty terrible error message as is.
